PaymentHighway/UI/Theme/DefaultTheme.swift

```diff
@@ -116,25 +121,36 @@ open class DefaultTheme : Theme {
-        self.borderWidth = defaulBorderWidth
-        self.textImages = defaultTextImages
-        self.textPaddingX = defaultTextPaddingX
-        self.primaryBackgroundColor = defaulBackgroundColor
-        self.secondaryBackgroundColor = defaulBackgroundColor
-        self.primaryForegroundColor = defaultPrimaryForegroundColor
-        self.primaryActiveForegroundColor = defaultPrimaryActiveForegroundColor
-        self.secondaryForegroundColor = defaultSecondaryForegroundColor
-        self.secondaryActiveForegroundColor = defaultSecondaryActiveForegroundColor
-        self.errorForegroundColor = defaultErrorForegroundColor
-        self.errorActiveForegroundColor = defaultErrorActiveForegroundColor
-        self.highlightColor = defaultHighlightColor
-        self.highlightDisableColor = defaultHighlightDisableColor
-        self.showKeyboard = defaultShowKeyboard
-        self.emphasisFont = UIFont.systemFont(ofSize: defaultFontSize+1, weight: .bold)
-        self.font = UIFont.systemFont(ofSize: defaultFontSize, weight: .regular)
-        self.expiryDatePicker = defaultExpiryDatePicker
+        placeholderAnimationDuration = defaultPlaceholderAnimationDuration
+        borderRadius = defaultRoundedBorderRadius
+        placeholderFontScale = defaultPlaceholderFontScale
+        borderWidth = defaultBorderWidth
+        textImageHeight = defaultTextImageHeight
+        textPaddingX = defaultTextPaddingX
+        barTintColor = defaultBarTintColor
+        primaryBackgroundColor = defaultBackgroundColor
+        secondaryBackgroundColor = defaultBackgroundColor
+        primaryForegroundColor = defaultPrimaryForegroundColor
+        primaryActiveForegroundColor = defaultPrimaryActiveForegroundColor
+        secondaryForegroundColor = defaultSecondaryForegroundColor
+        secondaryActiveForegroundColor = defaultSecondaryActiveForegroundColor
+        errorForegroundColor = defaultErrorForegroundColor
+        errorActiveForegroundColor = defaultErrorActiveForegroundColor
+        highlightColor = defaultHighlightColor
+        highlightDisableColor = defaultHighlightDisableColor
+        showKeyboard = defaultShowKeyboard
+        emphasisFont = UIFont.systemFont(ofSize: defaultFontSize+1, weight: .bold)
+        font = UIFont.systemFont(ofSize: defaultFontSize, weight: .regular)
+        expiryDatePicker = defaultExpiryDatePicker
+    }
+    
+    open func textImageView(textFieldType: TextFieldType, cardBrand: CardBrand? = nil) -> UIView? {
+        let iconImageView = UIImageView(frame: CGRect.zero)
+        iconImageView.image = UIImage(named: textFieldType.iconId(cardBrand: cardBrand),
+                                      in: Bundle(for: type(of: self)),
+                                      compatibleWith: nil)
+        iconImageView.frame = CGRect(x: 0, y: 0, width: textImageHeight, height: textImageHeight)
+        iconImageView.contentMode = UIViewContentMode.scaleAspectFit
```

Comment:
No need to write `UIViewContentMode` here explicitly?

Reply:
yes no need of it removed. thanks